compiler/rustc_codegen_ssa/src/base.rs

```diff
@@ -790,7 +788,6 @@ pub fn codegen_crate<B: ExtraBackendMethods>(
                         source: cgu.previous_work_product(tcx),
                     },
                 );
-                true
             }
         };
```

Comment:
This `;` can be removed, right?

Reply:
Yes. I'm surprised rustfmt didn't do that automatically. I have updated the code.